pages/api/link.js

```diff
@@ -0,0 +1,35 @@
+import Field from "../../models/fellow"
+import auth0 from "../../utils/auth"
+import dbConnect from "../../utils/db"
+
+export default async function link(req, res) {
```

Comment:
Could the contents of this function be wrapped in a try-catch block? Else an uncaught promise error could happen.